src/styles/components/header.scss

```diff
@@ -1,3 +1,7 @@
 .lbh-header__service-name {
   font-family: $govuk-font-family;
 }
+
+.govuk-heading-l {
+  margin-bottom: 10px !important;
+}
```

Comment:
I would add another class for doing this. This would mean anytime we use `.govuk-heading-l` it would have this overriding margin and we should try to avoid `!important` if possible